webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -248,15 +248,25 @@ public List<AtlasAccessorResponse> getAccessors(List<AtlasAccessorRequest> atlas
     }
 
     private AtlasEntityHeader getAtlasEntityHeader(String entityGuid, String entityId, String entityType) throws AtlasBaseException {
-        AtlasEntityHeader entityHeader;
+        AtlasEntityHeader entityHeader = null;
 
         if (StringUtils.isNotEmpty(entityGuid)) {
             AtlasEntityWithExtInfo ret = entitiesStore.getByIdWithoutAuthorization(entityGuid);
             entityHeader = new AtlasEntityHeader(ret.getEntity());
         } else if (StringUtils.isNotEmpty(entityId) && StringUtils.isNotEmpty(entityType)) {
             Map<String, Object> attributes = new HashMap<>();
             attributes.put(QUALIFIED_NAME, entityId);
-            entityHeader = new AtlasEntityHeader(entityType, attributes);
+            AtlasEntityType atlasEntityType = ensureEntityType(entityType);
+
+            try {
+                AtlasEntityWithExtInfo ret  = entitiesStore.getAtlasEntityWithExtInfoWithoutAuthorization(atlasEntityType,attributes,false,true);
```

Comment:
Shouldn't we pass the value of the isMinExtInfo param as "false"?

Reply:
passed isMinExtInfo as true